google-marketplace/README.md

```diff
@@ -0,0 +1,173 @@
+# Google Cloud Launcher
+
+## Overview
+
+Datadog provides infrastructure monitoring, application performance monitoring, and log management in a single-pane-of-glass view so teams can scale rapidly and maintain operational excellence.
+
+Installing the Datadog package via Google Cloud Launcher deploys the Datadog agent on every node in your Kubernetes cluster, and configures it with a secure, RBAC-based authentication and authorization model.
```

Comment:
r/`Datadog agent`/`Datadog Agent` (Agent should always be capitalised when speaking about the Datadog Agent)